packages/lib/src/components/internal/Await/Await.tsx

```diff
@@ -127,9 +140,13 @@ function Await(props: AwaitComponentProps) {
                 }
             };
 
+            // Reset 'loading' to ensure that it is changes to this value that are the main "engine" that drives this useEffect
+            setLoading(true);
+
+            // Create (another) interval to poll for a result
             setStoredTimeout(setTimeout(statusInterval, delay));
         }
-    }, [loading, timePassed, expired, completed]);
+    }, [loading, expired, completed]);
```

Comment:
Are you sure we don't need the `timePassed` here? We rely on it inside this useEffect , and its value gets updated often

Reply:
But I'm going to set it back - 'cos using `loading` as the drive has an odd effect on the UI